backend/app/assets/stylesheets/spree/backend/shared/_fonts.scss

```diff
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: 'Inter';
+  font-style:  normal;
+  font-weight: $font-weight-normal;
+  src: font-url("../../fonts/inter/Inter-Regular.woff2") format("woff2"),
+       font-url("../../fonts/inter/Inter-Regular.woff") format("woff");
+}
+
+@font-face {
+  font-family: 'Inter';
+  font-style:  normal;
+  font-weight: $font-weight-bold;
+  src: font-url("../../fonts/inter/Inter-SemiBold.woff2") format("woff2"),
+       font-url("../../fonts/inter/Inter-SemiBold.woff") format("woff");
```

Comment:
Hi @tvdeyen thank you for your precious suggestions!
I've moved the `fonts` path to `vendor/assets/` path. 
I cannot make it before, because I didn't know I should have added `config.assets.precompile << /\.(?:svg|eot|woff|ttf)\z/` to let Rails to compile font formats 😅

Reply:
👍